cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1539,21 +1539,29 @@ if(cuda OR tmva-gpu)
     endif()
 
     enable_language(CUDA)
-    
+
+    set(cuda ON CACHE BOOL "Found Cuda for TMVA GPU" FORCE)
+
     ### look for package CuDNN
-    find_package(CuDNN)
+    if (cudnn )
+      find_package(CuDNN)
 
-    if (CUDNN_FOUND)
-      message(STATUS "CuDNN library found: " ${CUDNN_LIBRARIES})
-    else()
-      message(STATUS "CuDNN library not found")
+      if (CUDNN_FOUND)
+	message(STATUS "CuDNN library found: " ${CUDNN_LIBRARIES})
+      	set(tmva-cudnn ON)
+      else()
+	message(STATUS "CuDNN library not found")
```

Comment:
Indentation. And please handle no CuDNN in the `fail-on-missing` case

Reply:
Sorry, I did not see in my editor, due to some hidden tabs. It will be fixed in the next  comit